Docker/README.md

```diff
@@ -872,21 +913,23 @@ As mentioned during the setup (below), this is convenient for a limited installa
 ![Owncloud sqlite setup](/Docker/img/owncloud_setup.png)
 
 In order to install owncloud on another database:
-   1. Wipe `config.php` to have the setup page proposed again by the application.
-   2. Add the `php-mysql` package to your Dockerfile in the relevant part.
-   3. Start the application but use `docker-compose up -d --build` to force the rebuild of the Dockerfile.
+   <!-- Not sure if the first item in the list is applicatible. I do not remember the file being present !-->
+   <!-- 1. Wipe `config.php` to have the setup page proposed again by the application. !-->
+   <!-- Changed the numbering of the remaining entries !-->
```

Comment:
Well you should have one. But maybe you don't because it was removed by a previous modification upper. See other comment.